koku/masu/database/trino_sql/openshift/cost_model/hourly_vm_core.sql

```diff
@@ -48,7 +48,11 @@ WITH
             vm_map.vm_cpu_request_cores AS vm_cpu_cores,
             DATE(vm_map.interval_start) AS interval_day,
             vm_map.vm_name AS vm_name,
-            count(vm_map.interval_start) AS vm_interval_hours
+            {% if use_fractional_hours %}
```

Comment:
For this specific metric, the customer is required to be on operator 4.0.0, and since this metric is not live yet, we can just go ahead and switch completely over to fractional hours here.